src/shared/components/case-list-filters/case-list-filters.component.html

```diff
@@ -0,0 +1,41 @@
+<ng-container *ngIf="jurisdictions">
```

Comment:
Case list filters should be reusing workbasket filters not copying it similar to search filters and search filters wrapper.

Reply:
Done